File: Encyption/JavaScript/CaesarCipher.js
const prompt = require("prompt-sync")();

let plainTextMessage = prompt("Enter a word: ");
let shiftAmount = parseInt(prompt("Enter shift number: "));

let encryptedMessage = "";

for (let position = 0; position < plainTextMessage.length; position++) {

    let currentCharacter = plainTextMessage[position];

    let encryptedCharacter = String.fromCharCode(
        currentCharacter.charCodeAt(0) + shiftAmount
    );

    encryptedMessage = encryptedMessage + encryptedCharacter;
}

console.log("Encrypted message: " + encryptedMessage);

File: Encyption/Java/CaesarCipher.java
import java.util.Scanner;

public class CaesarCipher {

    public static void main(String[] args) {

        Scanner inputReader = new Scanner(System.in);

        System.out.print("Enter a word to encrypt: ");

        String plainTextMessage = inputReader.nextLine();

        System.out.print("Enter encryption shift value: ");

        int shiftAmount = inputReader.nextInt();

        String encryptedMessage = "";

        for (int position = 0; position < plainTextMessage.length(); position++) {

            char currentCharacter = plainTextMessage.charAt(position);

            char encryptedCharacter = (char) (currentCharacter + shiftAmount);

            encryptedMessage = encryptedMessage + encryptedCharacter;
        }

        System.out.println("Encrypted message: " + encryptedMessage);

    }
}
